src/HDWallet.cpp

```diff
@@ -287,15 +283,18 @@ HDNode getMasterNode(const HDWallet& wallet, TWCurve curve) {
 const char* curveName(TWCurve curve) {
     switch (curve) {
     case TWCurveSECP256k1:
+    default:
```

Comment:
should we crash on default case? instead of returning `SECP256K1_NAME`

Reply:
Not sure what's best.  I'm reluctant to add unreachable/untestable lines of code; all 6 enum values are handled, so it's not possible to drive into separate default line.  Now I will delete the default line, though that may trigger compiler warning.

Reply:
Solved by introducing TWCurveNone value.